shuup/core/signal_handlers.py

```diff
@@ -39,10 +40,14 @@
 
 @receiver(post_migrate)
 def on_migrate(sender, **kwargs):
-    from .models import SupplierModule
+    from .models import OrderStatus, OrderStatusManager, SupplierModule
 
     SupplierModule.ensure_all_supplier_modules()
 
+    os_qs = OrderStatus.objects.filter(~Q(allowed_next_statuses=None))
+    if not os_qs:
+        OrderStatusManager().ensure_allowed_next_statuses()
```

Comment:
I think it is safe to run this every time 🤔

Reply:
Yes, it's sure. But every time we run this method we reset the allowed_next_statuses field of the predefined status. If the user changed them, we will overwrite them.

Reply:
so make `ensure_allowed_next_statuses` to not overwrite if they were already created, make it safe to run everytime